aws-ssm-maintenancewindowtask/resource-role.yaml

```diff
@@ -0,0 +1,38 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Description: >
+  This CloudFormation template creates a role assumed by CloudFormation
+  during CRUDL operations to mutate resources on behalf of the customer.
+
+Resources:
+  ExecutionRole:
+    Type: AWS::IAM::Role
+    Properties:
+      MaxSessionDuration: 8400
+      AssumeRolePolicyDocument:
+        Version: '2012-10-17'
+        Statement:
+          - Effect: Allow
+            Principal:
+              Service: resources.cloudformation.amazonaws.com
+            Action: sts:AssumeRole
+      Path: "/"
+      Policies:
+        - PolicyName: ResourceTypePolicy
+          PolicyDocument:
+            Version: '2012-10-17'
+            Statement:
+              - Effect: Allow
+                Action:
+                - "iam:PassRole"
+                - "ssm:DeregisterTaskFromMaintenanceWindow"
+                - "ssm:DescribeMaintenanceWindowTask"
+                - "ssm:DescribeMaintenanceWindowTasks"
+                - "ssm:GetMaintenanceWindow"
```

Comment:
Where is GetMaintenanceWindow being used

Reply:
When registerMaintenanceWindowTask, we first getMW from the windowStore and eventually emit the task creation events with window.